### ISSUE TITLE
stuck in "docker desktop is starting...."

### ISSUE THREAD

**Issue by jy1989** [docker/for-mac#3357]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID:  74816704-6636-4DA4-A85A-F69E539DFEDB/20181122080841

### Information

  - macOS Version: 10.14

these days when I start the docker from my mac, it stuck in docker desktop is starting.. 
I've tried to reset to factory defaults, but nothing help

![image](https://user-images.githubusercontent.com/1291756/48890300-71a3a580-ee73-11e8-934a-26e637154085.png)

**Comment by eweil505** [comment]
+1 for me

**Comment by beramb** [comment]
+1

**Comment by chiesennegs** [comment]
Same here, running 2.0.0.0-mac81 on Mojave 10.14.2. Doing a manual restart on Docker brings it into a normal "Docker Desktop is running" state almost immediately for me.

**Comment by davyzhang** [comment]
+1 here

**Comment by WBB-Badtaste** [comment]
+1 on Win 10

**Comment by jeepkd** [comment]
I solved this with force quitting the processes in Activity Manager. And then start the application again

**Comment by davyzhang** [comment]
roll back to an older version "Version 18.06.1-ce-mac73 (26764)", the problem has gone. I guess I will stick with that till this issue has been resolved.

**Comment by zp6827** [comment]
+1

**Comment by Asinox** [comment]
+1

**Comment by edbergavera** [comment]
I have the same exact issue.

```
Docker Version 2.0.1.0 (30090)
Channel: edge
Build: c8b6ca6945
```

**macos Mojave 10.14.2**

_Note: same issue with Stable channel._

**Comment by jarledrivenes** [comment]
![skjermbilde 2019-03-04 kl 18 57 17](https://user-images.githubusercontent.com/3584236/53752635-6539da80-3eaf-11e9-9ce0-45c3135232a8.png)

Experiencing the same problem....

**Comment by ozanonurtek** [comment]
+1 here, Version 2.0.0.3 (31259).

**Comment by seanwevans** [comment]
+1 win64 v2.0.0.3

**Comment by ekiliqua** [comment]
+1 Version 2.0.0.3

**Comment by shalu-007** [comment]
+1

**Comment by JtMotoX** [comment]
+1

**Comment by andrewjhinger** [comment]
+1

**Comment by nmenego** [comment]
+1

**Comment by tdingman** [comment]
+1

**Comment by daniel-marquez** [comment]
Not sure if this is just a workaround or just a way to fix the issue in OSX 10.14.6.

In my case, Docker decided to not work very early in the morning when I was about to start working.
Checking OSX Console, I saw that little time after trying to start Docker, the console displayed strange messages, with non usual characters.

I tried absolutely everything I saw on the net (reseting, reinstalling, switching to edge, downgrading to older versions... all combined with lots of OSX reboots) but has no success at all
I was about to reinstall the OS when decided to try something that didn't see on internet: restart OSX in safe mode.

After restarting in safe mode, I tried to run Docker and looks like it's not possible at all, but it gave me the chance to reset to factory defaults from there, and I accepted.

After rebooting again to normal mode, Docker started to work correctly.

Hope this may help others having he same issue.
